pkg/apis/certmanager/v1/types_issuer.go

```diff
@@ -209,6 +209,12 @@ type VaultIssuer struct {
 	// system root certificates are used to validate the TLS connection.
 	// +optional
 	CABundle []byte `json:"caBundle,omitempty"`
+
+	// If true, TLS requests against the Vault server are not verified
+	// for authenticity.  This allows one to issue certificates in a
+	// failure scenario, or when the Vault certificate has expired, for
+	// example.
+	SkipTLSVerify bool
```

Comment:
Could we use the same comments as in https://github.com/cert-manager/cert-manager/blob/master/internal/apis/acme/types_issuer.go#L52-L69?

Reply:
+ we would need to validate that only one of the 2 fields is set (CABundle OR SkipTLSVerify).
In the future we might have a SecretRef and ConfiMapRef field too from which we fetch the CABundle (see https://github.com/cert-manager/cert-manager/pull/6228).

Reply:
I've updated the comment as requested and rebased the PR.  I don't follow about validating only one is set.  Do you mean the Kubernetes API would reject the change?